src/components/list/searchList.module.scss

```diff
@@ -39,7 +39,18 @@
   justify-self: center;
 }
 
+.row {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+}
+
+.showOnMap {
```

Comment:
This rule is probably not really used?

Reply:
Nope, not really sure where I was going with this. I'll remove this.

Reply:
This has not been removed. Was it used after all?